regset/SPE1CASE2_REGSET.DATA

```diff
@@ -0,0 +1,264 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2015 Statoil
+-- Copyright (C) 2023 Equinor ASA
```

Comment:
I think we should change that to 2015-2023 Equinor ASA as the company was renamed.

Reply:
> I think we should change that to 2015-2023 Equinor ASA as the company was renamed.

Good point.  I've pushed an update to that effect.